styles/globals.css

```diff
@@ -1461,3 +1457,77 @@ textarea {
     margin-top: 10px;
   }
 }
+
```

Comment:
for date picker we would have separate CSS file... inside styles folder: `( datepicker.css)`

Reply:
ok